<launch>

    <!-- launch teleop xbox360 twist -->
    <arg name="twist_topic" default="/twist_controller/command" />
    <param name="joy_node/dev" type="str" value="/dev/input/js0" />
    <node pkg="joy" type="joy_node" name="joy_node" respawn="true" output="screen"/>
    <node pkg="morpheus_teleop" type="xbox360_effectorframe.py" name="xbox360_twist" output="screen">
        <param name="twist_topic" value="$(arg twist_topic)" />
    </node>

    <!-- tcp pose publisher -->
    <node pkg="morpheus_teleop" type="tcp_pose_publisher.py" name="tcp_pose_publisher" output="screen"/>

</launch>